Maestro.json

```diff
@@ -73,9 +73,9 @@
       "Arrivals": [
         {
           "FeederFix": "BOREE",
-          "ArrivalRegex": "/BOREE\\d[AP]/",
-          "Category": "Jet",
-          "AdditionalAircraftTypes": ["DH8D"],
+
+          "ArrivalRegex": "/BORE\\d[AP]/",
+          "AircraftType": "Jet",
```

Comment:
Merge conflict gone wrong?
`AircraftType` should now be called `Category`, and we still need the `AdditionalAircraftTypes` field.

Reply:
`AircraftType` will match an exact aircraft type, so you could use `"AircraftType": "DH8D"` for times that _only_ apply to DH8D.
`Category` is either `Jet` or `NonJet`. But you can use `AdditionalAircraftTypes` to whitelist other aircraft types.

This will match Jets and DH8D:
```json
"Category": "Jet",
"AdditionalAircraftTypes": ["DH8D"],
```

This will _only_ match DH8D:
```json
"AircraftType": "DH8D",
```

All of these fields are optional.

Reply:
I'm not a huge fan of the format. I'm open to suggestions if you've got any.

Reply:
Affirm!